src/common/darktable.h

```diff
@@ -528,6 +528,12 @@ static inline size_t dt_get_num_threads()
 #endif
 }
 
+static inline size_t dt_get_wanted_threads()
```

Comment:
shouldn't we change `dt_get_num_threads()` to return the number of threads for our OpenMP loop.

The returned value would be a `CLAMP(omp_get_num_procs(), 1, darktable.num_openmp_threads)`

and use that instead of `dt_get_wanted_threads()` ?

Reply:
> shouldn't we change `dt_get_num_threads()` to return the number of threads for our OpenMP loop.

Absolutely correct.

I just checked and there might be bugs related to this as the function gets the number of cores and not threads but often we require just threads.